bin/test-with-real-projects.sh

```diff
@@ -22,8 +22,8 @@ phpunit)
 	;;
 
 collections)
-	git clone --depth=1 git@github.com:muglug/collections.git
-	cd collections
+	git clone --depth=1 git@github.com:psalm/endtoend-test-collections.git
```

Comment:
those two could probably removed. AFAIK, they're not in CI anymore...

Reply:
`collections` is used:

https://github.com/vimeo/psalm/blob/aa7e400908833b10c0333861f86cd48c510b60eb/.circleci/config.yml#L68-L76